kill_list.md

```diff
@@ -5,3 +5,5 @@ Kill List
 * Unformatted code
```

Comment:
Awesome job! Go ahead and merge this back into master.

Reply:
Aw man! Thanks so much! This is great!